opensearch_dashboards.json

```diff
@@ -16,5 +16,7 @@
     "opensearchDashboardsUtils"
   ],
   "server": true,
-  "ui": true
+  "ui": true,
+  "supportedOSDataSourceVersions": ">=2.13.0",
```

Comment:
Saw this comment in this other PR https://github.com/opensearch-project/dashboards-notifications/pull/221#discussion_r1680328882. Should this version be adjusted?

Reply:
i have updated the description, why we need >=2.13